static/src/stylesheets/module/onward-garnett/_feedback.scss

```diff
@@ -48,6 +48,33 @@
     width: 80%;
 }
 
+.feedback__alertbox {
+    background-color: $news-garnett-main-1;
+    color: #ffffff;
+    font-size: 1.2em;
+    padding: .5em;
+    line-height: 1.7em;
+    margin-bottom: 2em;
+}
+
+.feedback__exclamation {
+    margin-right: .5em;
+    border: 2px solid #ffe0e0;
+    border-radius: 99px;
+    width: 1.5em;
+    height: 1.5em;
+    text-align: center;
+    display: inline-block;
+    font-weight: bold;
+    line-height: 1.7em;
+}
+
+.feedback__alert {
+    margin-top: 1em;
+    border-top: solid 1px #f97070;
```

Comment:
@zeftilldeath what's the situation with colours following https://github.com/guardian/frontend/pull/19968, should they all be derived from variables?

Reply:
Ideally yes! There's enough in there to use. @MatthewJWalls I could do a quick mockup to get this in line with our usual styles?

Reply:
If you have time that'd be great, but I'm fine muddling an attempt if not